crates/core/src/ibc.rs

```diff
@@ -359,7 +359,10 @@ impl TryFrom<RawIbcEvent> for IbcEvent {
 /// denom.
 pub fn is_ibc_denom(denom: impl AsRef<str>) -> Option<(TracePath, String)> {
     let prefixed_denom = PrefixedDenom::from_str(denom.as_ref()).ok()?;
-    if prefixed_denom.trace_path.is_empty() {
+    if prefixed_denom.trace_path.is_empty()
+        || prefixed_denom.base_denom.to_string().contains('/')
```

Comment:
nit: avoid converting the `base_denom` to a string twice, and return this binding instead of calling `.to_string()` again at the end
```suggestion
    let base_denom = prefixed_denom.base_denom.to_string();
    if prefixed_denom.trace_path.is_empty() || base_denom.contains('/')
```

Reply:
Thanks! Fixed at [babaeab](https://github.com/anoma/namada/pull/3030/commits/babaeabe57334f981a20149ce37a5fac925efbe1)